java/semantickernel-api/src/main/java/com/microsoft/semantickernel/orchestration/ContextVariables.java

```diff
@@ -61,35 +77,34 @@ interface Builder extends SemanticKernelBuilder<ContextVariables> {
          * @param content Entry to place in the "input" slot
          * @return an instantiation of ContextVariables
          */
-        Builder withInput(String content);
+        <T> Builder withInput(ContextVariable<T> content);
```

Comment:
Not sure how the generic type helps here. I think ContextVariable<?> would be fine.